README.md

```diff
@@ -74,10 +74,10 @@ fetch('<issuer>/.well-known/trust-token', {
 This API will invoke the [Privacy Pass](https://privacypass.github.io) Issuance protocol:
 
 *   Generate a set of nonces.
-*   Blind them and attach them to the HTTP request
+*   Blind them and attach them (in a Sec-Trust-Tokens header) to the HTTP request
```

Comment:
nit: Probably should be Sec-Trust-Token to be consistent with all the other headers we're using in this doc (and I think Trust-Token is more consistent with the  Cookie header?

Reply:
Summarizing offline discussion:
- Having one unified issuance/redemption request/response header makes layering a bit easier because lower-level code might just want to look out for all pertinent headers, and this is simpler with only one header to worry about
- No obvious blocking downsides to having one header, although it makes error handling marginally harder (with two headers, if you get a redemption header at issuance time or v.v., you know something's wrong)

-> Changed the issuance and redemption headers to both be called Sec-Trust-Token